kotlin/kotlin-quiz.md

```diff
@@ -117,15 +117,15 @@ when (die) {
 }
 ```
 
-- [ ] `3,4,5,6`
+- [x] `3,4,5,6`
 - [ ] `in 3..6`
```

Comment:
```suggestion
- [x ] `in 3..6`
```

Reply:
@LeonardoJaques we can keep both answers but if you have an explanation that is 100% true then go ahead and put it here. We can even put your reference below the @kariot answer! I hope you guys agree, if not I am ready for any other ideas!

Reply:
@Ebazhanov For me its okay, you really can use both.
 I use the official site as reference -> https://kotlinlang.org/docs/ranges.html

Reply:
ok, great! Thanks for feedback @roplayer! I will merge the changes from @kariot and later push to master your suggestion with reference link.